portability-api-launcher/build.gradle

```diff
@@ -0,0 +1,3 @@
+dependencies {
+    compile("org.slf4j:slf4j-log4j12:${slf4jVersion}")
```

Comment:
I'm wondering if we need this in the api-launcher API launcher build file.  Isn't it only setup to work in the TransferWorker for now?

Reply:
I needed it in there for ExtensionContext.  I agree it feels weird in there.